### PR TITLE
Add actor2D

### PR DESCRIPTION
**Pull request by chakravarthi589**

### CODE REVIEW

File: src/Java/Images/Actor2D.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Description
```
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: Should be:
### Description

File: src/Java/Images/Actor2D.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Example to demonstrate a actor that draws 2D data.
+
+The object vtkActor2D is similar to vtkActor, but it is made to be used with two dimensional images and annotation. 
+
+vtkActor2D object has a position but does not use a transformation matrix like vtkActor and it has a reference to a vtkMapper2D object which does the rendering.
```
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: Add a trailing blank line

File: src/Java/Images/Actor2D.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Description
+
```
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: This should be:
`### Description`

File: src/Java/Images/Actor2D.md (hunk 2 above)
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: Always add a trailing blank line.

File: src/Java/Images/Actor2D.md (hunk 3 above)
Comment (chakravarthi589) [reply src/Java/Images/Actor2D.md]: Please check now

File: src/Java/Images/Actor2D.md (hunk 2 above)
Comment (chakravarthi589) [reply src/Java/Images/Actor2D.md]: I have added now.

File: src/Java/Images/Actor2D.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Description
+### Description
 
 Example to demonstrate a actor that draws 2D data.
```
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: Example to demonstrate an actor that draws 2D data.

File: src/Java/Images/Actor2D.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Description
+
+Example to demonstrate a actor that draws 2D data.
+
+The object vtkActor2D is similar to vtkActor, but it is made to be used with two dimensional images and annotation. 
+
+vtkActor2D object has a position but does not use a transformation matrix like vtkActor and it has a reference to a vtkMapper2D object which does the rendering.
+
+
```
Comment (ajpmaclean) [inline src/Java/Images/Actor2D.md]: Looks to be an extra trailing blank line here, leave just one.

File: src/Java/Images/Actor2D.md (hunk 4 above)
Comment (chakravarthi589) [reply src/Java/Images/Actor2D.md]: I have corrected.

File: src/Java/Images/Actor2D.md (hunk 5 above)
Comment (chakravarthi589) [reply src/Java/Images/Actor2D.md]: is it ok now?

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**